frontend/src/app/organization/organization.component.ts

```diff
@@ -35,7 +35,7 @@ export class OrganizationComponent implements OnInit {
 
     this.activatedRoute.params.subscribe(
       params => {
-        this.organizationService.getOrganization(params.organizationId);
+        this.organizationService.getOrganization(params.organizationId).subscribe();
```

Comment:
what's the reason of this subscribe?

Reply:
@magul The reason is that the pattern that we use for exposing observables in OrganizationService has been 'broken' by Julia and this is the side effect. In my opinion we shouldn't allow this to happen. Please check my comment on line 50 in organization-create.component.ts for more details on that.